### PR TITLE
[FIX] Magic/Wild Mushroom Hoarder Limit Front End

### PR DESCRIPTION
**Pull request by Droidze**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/lib/processEvent.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ export const MAX_ITEMS: Inventory = {
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
   "Wild Mushroom": new Decimal("100"),
+  "Magic Mushroom": new Decimal("100"),
```
Comment (eliasSFL) [inline src/features/game/lib/processEvent.ts:126]: We usually try to put the FE Hoard limit to be lower than the BE, could you adjust this to 75?

**Review by eliasSFL — CHANGES_REQUESTED** [review]

File: src/features/game/lib/processEvent.ts (hunk 1 above)
Comment (Droidze) [reply src/features/game/lib/processEvent.ts:126]: Is this situation the same for Wild Mushrooms? how much is their hoard limit on the backend?

Comment (eliasSFL) [reply src/features/game/lib/processEvent.ts:126]: > @eliasSFL How much is the wild mushrooms hoard limit on the backend?

@Droidze 100

File: src/features/game/lib/processEvent.ts [hunk 2]
```diff
@@ -122,8 +122,8 @@ export const MAX_ITEMS: Inventory = {
   Iron: new Decimal("800"),
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
-  "Wild Mushroom": new Decimal("100"),
-  "Magic Mushroom": new Decimal("100"),
+  "Wild Mushroom": new Decimal("75"),
```
Comment (Droidze) [inline src/features/game/lib/processEvent.ts:125]: @eliasSFL OK, the wild mushrooms' limit was also decreased, following the same rule.

Comment (eliasSFL) [reply src/features/game/lib/processEvent.ts:125]: @Droidze You can leave the wild mushroom at 100. cuz the BE for wild mushroom is at 150

**Review by eliasSFL — APPROVED** [review]

File: src/features/game/lib/processEvent.ts [hunk 3]
```diff
@@ -122,7 +122,7 @@ export const MAX_ITEMS: Inventory = {
   Iron: new Decimal("800"),
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
-  "Wild Mushroom": new Decimal("75"),
+  "Wild Mushroom": new Decimal("100"),
```
Comment (Droidze) [inline src/features/game/lib/processEvent.ts:125]: @eliasSFL, why did you revert my changes? I see a contradiction here. If it matters to keep a tolerance between FE and BE, the wild mushrooms should also follow this rule.
It's the same situation that players are experiencing with both kinds of mushrooms: they receive a sync message after getting the 100th wild mushroom and an error message after getting the 100th magic mushroom between two syncs. Your idea is to leave things as they are, which is also inconsistent with the rule that says to keep a tolerance between FE and BE. Even if we want to follow your idea, we should leave the magic mushrooms as they are too.

File: src/features/game/lib/processEvent.ts (hunk 2 above)
Comment (Droidze) [reply src/features/game/lib/processEvent.ts]: @eliasSFL I'm confused now, you said in your previous message wild mushrooms' hoard limit is 100 in the BE, so I've made changes for both of them. 
image

**Pull request merged**